packages/landingpage-new/components/shared/navigationMenu.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import styles from './navigationMenu.module.scss';
+
+export function NavigationMenu() {
+  const [activeSection, setActiveSection] = useState('preparation');
```

Comment:
Instead of using plain strings I would suggest introducing an enum to handle all the sections.

e.g.:
```ts
enum Sections {
  PREPARATIONS = 'preparations',
  PROPERTIES = 'properties',
  // ...
}
```